metadata-ingestion/src/datahub/api/entities/datajob/datajob.py

```diff
@@ -5,6 +5,7 @@
 import datahub.emitter.mce_builder as builder
 from datahub.emitter.generic_emitter import Emitter
 from datahub.emitter.mcp import MetadataChangeProposalWrapper
+from datahub.metadata._schema_classes import DataPlatformInstanceClass
```

Comment:
```suggestion
from datahub.metadata.schema_classes import DataPlatformInstanceClass
```

To fix the lint issue:

```
FAILED tests/unit/test_packages.py::test_check_import_paths - ValueError: Disallowed import found in /home/runner/work/datahub/datahub/metadata-ingestion/src/datahub/api/entities/datajob/datajob.py: `from datahub.metadata._schema_classes import DataPlatformInstanceClass`. Import from datahub.metadata.schema_classes instead.
```

Reply:
Arf, fixed, thanks!